tests/test-definition.yaml

```diff
@@ -19,13 +19,13 @@ dimensions:
       - 3.1.3
       # Alternatively, if you want to use a custom image, append a comma and the full image name to the product version
       # as in the example below.
-      # - 3.1.3,docker.stackable.tech/sandbox/hive:3.1.3-stackable0.0.0-dev
+      - 4.0.0,docker.stackable.tech/sandbox/hive:4.0.0-stackable0.0.0-dev
```

Comment:
TODO: switch to stackable image once built

Reply:
Done, waiting for https://github.com/stackabletech/docker-images/actions/runs/10680049849